src/couch_stats/CSRT.md

```diff
@@ -0,0 +1 @@
+# Couch Stats Resource Tracker (CSRT)
```

Comment:
We should either add a readme file or not have one at all, just the title is probably not worth having.